coulttxml

```diff
@@ -78,23 +99,28 @@ system.addForce(CoulTT)
 
 def main():
     if len(sys.argv) < 3:
-        print("Usage: coulttxml field-p.xml config-p.pdb")
+        print("Usage: coulttxml field-p.xml config-p.pdb/config.mmcif")
```

Comment:
The standard notation for an option would be [config-p.pdb | config-p.mmcif]
Aren't we missing the '-p'?